SPLASH/src/game/states/GameState.cpp

```diff
@@ -1081,8 +1083,8 @@ void GameState::createLevel(Shader* shader, Model* boundingBoxModel) {
 
 	// Create the level generator system and put it into the datatype.
 	auto map = ECS::Instance()->createEntity("Map");
-	map->addComponent<MapComponent>();
-
+	map->addComponent<MapComponent>(NWrapperSingleton::getInstance().getSeed());
+	Logger::Log(std::to_string(NWrapperSingleton::getInstance().getSeed()));
```

Comment:
I would remove this logging or add a string before the seed to make it clear what it is. For example `Generating map of size x X y with seed 1337`.